Helper/Hook.php

```diff
@@ -35,6 +35,8 @@
  */
 class Hook extends AbstractHelper
 {
+    public static $fromBolt = false;
```

Comment:
I'm not very familiar with how php/magento works. Is it safe to use static variables for request-scoped variable? ie., if magento get two requests at the same time, is this variable scoped to one request?

Reply:
No, this variable is scoped to each request.